databricks_cli/configure/provider.py

```diff
@@ -28,13 +28,13 @@
 
 from databricks_cli.utils import InvalidConfigurationError
 
-
```

Comment:
Is this by mistake? I think PEP8 says there should be 2 lines between imports and any other code blocks.

Reply:
Fixed.